pkg/resource/claim_binding_reconciler.go

```diff
@@ -202,16 +201,19 @@ func defaultCRManaged(m manager.Manager) crManaged {
 		ManagedConfigurator:         NewObjectMetaConfigurator(m.GetScheme()),
 		ManagedCreator:              NewAPIManagedCreator(m.GetClient(), m.GetScheme()),
 		ManagedConnectionPropagator: NewAPIManagedConnectionPropagator(m.GetClient(), m.GetScheme()),
-		Binder:                      NewAPIBinder(m.GetClient(), m.GetScheme()),
 	}
 }
 
 type crClaim struct {
 	ClaimFinalizer
+	Binder
```

Comment:
I like that change. It sounds to me binding act belongs to claim, so, it makes sense for claim to `crClaim` manage it.